app/utils/display.py

```diff
@@ -184,7 +184,7 @@ def create_event_list_from_df(api_events):
             id={"type": "event-button", "index": event["id"]},
             children=[
                 html.Div(
-                    f"{event['device_name']}",
+                    f"{event['device_login'][:-2].replace('_', ' ') + ' - ' + str(int(event['device_azimuth'])) + '°'}",
```

Comment:
better to use f-string feature since you started with it:
```python
f"{event['device_login'][:-2].replace('_', ' ')} - {int(event['device_azimuth'])}°"
```